model/vm.rb

```diff
@@ -220,7 +202,10 @@ def params_json(swap_size_bytes)
       "ndp_needed" => vm_host.ndp_needed,
       "storage_volumes" => storage_volumes,
       "swap_size_bytes" => swap_size_bytes,
-      "pci_devices" => pci_devices.map { [_1.slot, _1.iommu_group] }
+      "pci_devices" => pci_devices.map { [_1.slot, _1.iommu_group] },
+      "slice_name" => vm_host_slice.nil? ? "system.slice" : vm_host_slice.inhost_name,
+      "cpu_percent_limit" => cpu_percent_limit.nil? ? 0 : cpu_percent_limit,
+      "cpu_burst_percent_limit" => cpu_burst_percent_limit.nil? ? 0 : cpu_burst_percent_limit
```

Comment:
```suggestion
      "slice_name" => vm_host_slice&.inhost_name || "system.slice",
      "cpu_percent_limit" => cpu_percent_limit || 0,
      "cpu_burst_percent_limit" => cpu_burst_percent_limit || 0
```

Same, just more idiomatic.

Reply:
OK, done.